apkid/rules/dex/packers.yara

```diff
@@ -70,3 +70,25 @@ rule medusah_appsolid_dex : packer
   condition:
     is_dex and $loader and $main_activity
 }
+
+rule apkguard : packer
+{
+  meta:
+    description = "ApkGuard"
+    info = "http://apkguard.io/"
+    // "Base64.decode("UEsDBB..."
```

Comment:
What's this comment for?

Reply:
Removed but it was there to see if we could detect this string in Dalvik code. To be more precise actually.